package.json

```diff
@@ -1,18 +1,26 @@
 {
```

Comment:
Are you running `npm init` or is VS Code being weird when you run `npm install`?